lib/cloudinary/migrator.rb

```diff
@@ -299,7 +299,9 @@ def start
   
   def debug(message)
     if @debug
-      $stderr.print "#{Time.now} Cloudinary::Migrator #{message}\n"
+      mutex.synchronize{

```

Comment:
New line flushes the io stream, so this seems redundant (although not much harmful).
Please refer to my comment in the related issue.